agent/tool-scripts/unittests

```diff
@@ -76,9 +76,10 @@ if [[ ${?} -ne 0 ]]; then
     echo "ERROR: failed to create util-scripts directory" >&2
     exit 1
 fi
-cp ${_tdir}/../util-scripts/require-rpm ${_optdir}/util-scripts/ 2> /dev/null
+printf -- "#!/bin/bash\n\n_ver=\"\${2}\"\nif [[ ! -z \"\${_ver}\" ]]; then _ver=\"-\${_ver}\"; fi\necho \"\${1}\${_ver}\"\nexit 0\n" > ${_optdir}/util-scripts/require-rpm
```

Comment:
Agreed, and there are two others just like it.  =)

Reply:
I am tempted to fix this as suggested, except I don't want to change the other ones that are doing the same thing, and want to shed all this entirely for a re-written Python3 code base.

Reply:
From what you say, there are in fact (at least...) __six__ places where we have separate instantiations of the `require-rpm` script...when it's time to move to a common mocking approach, how are we going to find them all??

I feel like the code change here, which changes from an external file to a dynamically generated one, is hiding the problem.  (And, I don't really see the benefit (in this case, especially) of generating the file over copying an existing one.)

Nevertheless, if you are confident that converting to Python will make the situation apparent as well as providing a well-grounded approach to addressing it, then I'm OK with this.  (Otherwise, please make it an external file where `find` and `grep` can find it....)

Reply:
I am desperately trying to avoid doing something different from the previous code or fixing the previous code, but this is ugly.  I'll noodle on a more elegant solution.

Reply:
I think we want to have a shared mock directory for all the legacy agent tests which is leveraged by each of them.

I'll track this is an issue for the future.

Reply:
Tracked in #2224.